spec/controllers/items_controller_spec.rb

```diff
@@ -253,16 +255,18 @@
     it 'should have dor-services fetch a file from the workspace' do
       allow(@item).to receive(:get_file).and_return('abc')
       expect(@item).to receive(:get_file)
+      allow(Time).to receive(:now).and_return(Time.parse 'Mon, 30 Nov 2015 20:19:43 UTC')
```

Comment:
Do not use `Time.parse` without zone. Use one of `Time.zone.parse`, `Time.parse.current`, `Time.parse.in_time_zone`, `Time.parse.utc`, `Time.parse.getlocal`, `Time.parse.iso8601`, `Time.parse.jisx0301`, `Time.parse.rfc3339`, `Time.parse.to_i`, `Time.parse.to_f` instead.